premis_event_service/views.py

```diff
@@ -575,15 +574,13 @@ def app_event(request, identifier=None):
             except FieldError:
                 # If order_by fails, revert to natural order.
                 events = unordered_events
-        debug_list = []
-        endTime = datetime.now()
         if request.GET:
             page = int(request.GET['page']) if request.GET.get('page') else 1
         else:
             page = 1
         try:
             atomFeed = makeObjectFeed(
-                paginator=Paginator(events, 20),
+                paginator=Paginator(events, 200),
```

Comment:
No, sorry. It's a mistake. I hard-coded it to make sure the paginator still worked properly, intending to either change it or set it to a new constant w/ the same value. Fix coming momentarily.